apps/synapse-portal-framework/src/pages/FileEntityPage/SynapseFileEntityPageProperties.tsx

```diff
@@ -158,14 +158,16 @@ const SynapseFileEntityPageProperties = ({
               storageLocationName={fileLocationName}
             />
           ))}
-          <tr>
-            <td style={{ width: labelWidth }}>
-              <Box sx={labelSxProps}>In Synapse Project</Box>
-            </td>
-            <td>
-              <EntityLink entity={projectEntity!} />
```

Comment:
Cannot assume this can be accessed by the current user

Reply:
Good call